### DISCUSSION TITLE
For auto resume of music after announcement

### DISCUSSION THREAD

**Discussion by casperjjordaan** [music-assistant/support#487] [Q&A]
Good day HA folk.

I need to ask, how can I setup MA to continue streaming tune-in after the stream has been interrupted by a announcement like "Somebodies at the door"? Because it does not auto resume my online radio station.

Thanks guys!

**Comment by marcelveldt** [comment]
Make sure that you send the announcement to the "mass" player and this will be handled for you.

**Reply by casperjjordaan** [reply]
You mean the primary group of speakers?

Thanks buddy, you're a legend!

**Reply by smiller171** [reply]
I still don't seem to be able to get this to work.

When I use the `tts.cloud_say` service the music doesn't resume, and when I trigger TTS from the media browser there's a delay, then partial tts audio, then a big pause before eventually resuming playback. Also I don't know how to trigger announcements this way from automations.

**Comment by OzGav** [comment]
Relates to v1
